website/docs/reference/exposure-properties.md

```diff
@@ -101,3 +104,30 @@ exposures:
 ```
 
 </File>
+
+#### Project-level configs
+
+You can define project-level configs for exposures in the `dbt_project.yml` file under the `exposures:` key using the `+` prefix:
+
+<File name="dbt_project.yml">
+
+```yml
+name: 'project_name'
+version: '1.0.0'
+config-version: 2
+
+.... rest of dbt_project.yml 
+
+exposures:
+  +enabled: true
+  +tags: ['docs']
+  +meta:
+    owner_team: analytics
```

Comment:
I think let's cut these - we can't guarantee that `tags` + `meta` propagate as expected right now